datalad/distribution/create_sibling.py

```diff
@@ -163,6 +157,28 @@ def _create_dataset_sibling(
                 lgr.info(_msg + " Skipping")
                 return
             elif existing == 'replace':
+                remove = False
+                if path_children:
+                    has_git = '.git' in path_children
+                    _msg_stats = _msg \
+                                 + " It is %sa git repository and has %d files/dirs." % (
+                                     "" if has_git else "not ", len(path_children)
+                                 )
+                    from datalad.ui import ui
+
+                    if ui.is_interactive:
+                        remove = ui.yesno(
+                            "Do you really want to remove it?",
+                            title=_msg_stats,
+                            default=False
+                        )
+                    else:
+                        remove = True
```

Comment:
Hmm, this is my main concern with this PR.  To me it seems like it'd be better to just make the replace-functionality interactive-only.

Reply:
ok, will do. better be safe than sorry they say ;-)